### PR TITLE
Changelog for 0.11.0

### PR DESCRIPTION
**Pull request by davkal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,82 @@
...
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
+	[#650](https://github.com/weaveworks/scope/pull/650)
+  [#735](https://github.com/weaveworks/scope/pull/735)
+  [#726](https://github.com/weaveworks/scope/pull/726)
+- Added `scope version` command
+	[#750](https://github.com/weaveworks/scope/pull/750)
+- Various CPU usage gains for probe
```
Comment (tomwilkie) [inline CHANGELOG.md]: s/gains/reductions/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+	[#742](https://github.com/weaveworks/scope/pull/742)
+	[#741](https://github.com/weaveworks/scope/pull/741)
+	[#737](https://github.com/weaveworks/scope/pull/737)
+- Show hostname of app you are connected to
+	[#707](https://github.com/weaveworks/scope/pull/707)
+- Add host memory and CPU usage metrics
+	[#711](https://github.com/weaveworks/scope/pull/711)
+- Add json support to app POST /api/report
+	[#722](https://github.com/weaveworks/scope/pull/722)
+- Add basic security warning.
```
Comment (tomwilkie) [inline CHANGELOG.md]: Maybe add a separate section for docs?  Not really a feature.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+	[#703](https://github.com/weaveworks/scope/pull/703)
+- Update the docker version we embed into the scope image to 1.6.2 in sync with weave 1.3 changes.
+	[#702](https://github.com/weaveworks/scope/pull/702)
+- Show a spinner while node details are loading
+	[#691](https://github.com/weaveworks/scope/pull/691)
+- Procedural colors for nodes
```
Comment (tomwilkie) [inline CHANGELOG.md]: This probably need a better description - "Make node colorrs deterministic based on node name."

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+	[#694](https://github.com/weaveworks/scope/pull/694)
+- Mitigate one-line-of-nodes layouts
```
Comment (tomwilkie) [inline CHANGELOG.md]: Move to bug fixes and describe it more?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,82 @@
...
+	[#679](https://github.com/weaveworks/scope/pull/679)
+- Add basic kubernetes how-to to the readme
```
Comment (tomwilkie) [inline CHANGELOG.md]: Move to docs section.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,82 @@
+## Release 0.11.0
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
```
Comment (tomwilkie) [inline CHANGELOG.md]: "Add a terminal to the UI with the ability to `attach` to, or `exec` a shell in, a Docker container."

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,82 @@
...
+  [#735](https://github.com/weaveworks/scope/pull/735)
+  [#726](https://github.com/weaveworks/scope/pull/726)
+- Added `scope version` command
+	[#750](https://github.com/weaveworks/scope/pull/750)
+- Various CPU usage gains for probe
+	[#742](https://github.com/weaveworks/scope/pull/742)
+	[#741](https://github.com/weaveworks/scope/pull/741)
+	[#737](https://github.com/weaveworks/scope/pull/737)
+- Show hostname of app you are connected to
+	[#707](https://github.com/weaveworks/scope/pull/707)
```
Comment (tomwilkie) [inline CHANGELOG.md]: ... in the bottom-right of the UI.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,82 @@
...
+- Add host memory and CPU usage metrics
```
Comment (tomwilkie) [inline CHANGELOG.md]: ... to the details panel.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,82 @@
...
+- Procedural colors for nodes
+	[#694](https://github.com/weaveworks/scope/pull/694)
+- Mitigate one-line-of-nodes layouts
+	[#679](https://github.com/weaveworks/scope/pull/679)
+- Add basic kubernetes how-to to the readme
+	[#669](https://github.com/weaveworks/scope/pull/669)
+
+Bug fixes
+
+- Document debug options.
```
Comment (tomwilkie) [inline CHANGELOG.md]: Move to documentation section?  Also append "for developers".

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,82 @@
...
+	[#723](https://github.com/weaveworks/scope/pull/723)
+- Filter nodes that are only connected to themselves.
```
Comment (tomwilkie) [inline CHANGELOG.md]: Prepend "When filtering unconnected nodes in the processes view, also "

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,82 @@
...
+	[#706](https://github.com/weaveworks/scope/pull/706)
+- Merge image metadata into container metadata so image labels can be used to hide containers.
```
Comment (tomwilkie) [inline CHANGELOG.md]: "Correctly hide container based on docker labels on the container image."

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,82 @@
...
+	[#705](https://github.com/weaveworks/scope/pull/705)
+- Dont send topology options when requesting node details
```
Comment (tomwilkie) [inline CHANGELOG.md]: Merge with one below (#701)

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,82 @@
...
+	[#704](https://github.com/weaveworks/scope/pull/704)
+- Don't apply filters to node endpoints & update details panel logic
```
Comment (tomwilkie) [inline CHANGELOG.md]: Show details for stopped container in the default view, by not applying filters to the node details endpoints.

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,82 @@
...
+- Dont treat missing node as UI error
+	[#677](https://github.com/weaveworks/scope/pull/677)
+- Unset previous details when deselecting a node
+	[#675](https://github.com/weaveworks/scope/pull/675)
+- Add x to close details panel again
+	[#673](https://github.com/weaveworks/scope/pull/673)
+
+Build improvements and cleanup
+
+- Don't go get weave, git clone it.
```
Comment (tomwilkie) [inline CHANGELOG.md]: ... so weave build errors don't affect Scope.

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,82 @@
...
+  [#743](https://github.com/weaveworks/scope/pull/743)
+- Merge probe and app binaries, run with ./scope probe or ./scope app.
```
Comment (tomwilkie) [inline CHANGELOG.md]: "Reduce image size and build time by merging scope probe and app binaries."

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,3 +1,82 @@
...
+	[#732](https://github.com/weaveworks/scope/pull/732)
+- Cleaning up some dead code around edges and edgemetadata
+	[#730](https://github.com/weaveworks/scope/pull/730)
+- Make make build ui
```
Comment (tomwilkie) [inline CHANGELOG.md]: "Make `make` build the UI"

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,3 +1,82 @@
...
+	[#728](https://github.com/weaveworks/scope/pull/728)
+- Omit controls field from json if emtpy.
+	[#725](https://github.com/weaveworks/scope/pull/725)
+- JS to ES2015
+	[#712](https://github.com/weaveworks/scope/pull/712)
+- Add 'getting help' section and update screenshot
```
Comment (tomwilkie) [inline CHANGELOG.md]: Move to documentation section.

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,3 +1,82 @@
+## Release 0.11.0
+
+New features:
+
+- UI Terminal to attach to Docker containers and see their output or `EXEC` a shell on them
+	[#650](https://github.com/weaveworks/scope/pull/650)
```
Comment (peterbourgon) [inline CHANGELOG.md]: Tabs v. spaces! :)

**Comment by tomwilkie** [comment]
LGTM

**Pull request merged**